lib/SlackEventHandler.js

```diff
@@ -161,7 +177,24 @@ SlackEventHandler.prototype.handleMessageEvent = function (params) {
         channel_id: params.event.channel
     });
 
-    const processMsg = msg.text || msg.subtype === 'message_deleted' || msg.files != undefined || msg.subtype === 'message_changed';
+    // Handle events with attachments like bot messages.
+    if (params.event.type === "message" && params.event.attachments) {
+        log.warn(params.event.attachments);
+        for (var attachment of params.event.attachments) {
+            msg.text = attachment.fallback;
+
+            msg = await this.doChannelUserReplacements(msg, token);
+            await room.onSlackMessage(msg);
+        }
+        if (params.event.text != '') {
```

Comment:
Can't help but wonder if we could invert this
```suggestion
        if (params.event.text == '') {
            return;
        }
        msg.text = params.event.text;
```